swagger.yaml

```diff
@@ -28,17 +31,35 @@ paths:
       - Domains
       summary: Get All Domains
       operationId: GetAllDomains
-      description: Retrieves all of the domain configuration profiles stored in the database.  Will not return the provisioning certificate or certificate password to protect the privacy of these assets.
+      description: Retrieves all of the domain configuration profiles stored in the database.  Will not return the provisioning certificate or certificate password to protect the privacy of these assets
+      parameters:
+        - in: query
+          name: $skip
+          schema:
+            type: integer
+          description: The number of items to skip before starting to collect the result set
+        - in: query
+          name: $top
+          schema:
+            type: integer
+          description: The numbers of items to return
```

Comment:
might want to change to "The number of items to return"

Reply:
done